pypeit/spectrographs/spectrograph.py

```diff
@@ -661,8 +661,12 @@ def get_meta_value(self, inp, meta_key, required=False, ignore_bad_header=False,
 
         # Deal with 'special' cases
         if meta_key in ['ra', 'dec'] and value is not None:
-            ra, dec = meta.convert_radec(self.get_meta_value(headarr, 'ra', no_fussing=True),
-                                self.get_meta_value(headarr, 'dec', no_fussing=True))
+            try:
```

Comment:
Let's avoid this try/except.  What is causing it?

Reply:
Meta data parsing always crashes when headers are bad. This happens often. Added a TODO on this.